### DISCUSSION TITLE
Add DNAME record support

### DISCUSSION THREAD

**Discussion by T145** [projectdiscovery/dnsx#134] [General]
### Please describe your feature request:

Add a `-dname` flag to allow DNAME record queries.

### Describe the use case of this feature:

DNAME records hold a tree of subdomains, which can be useful information.

**Comment by ehsandeep** [comment]
@T145 do you have an example host using "dname" record to share?

**Reply by ehsandeep** [reply]
moved to discussion due to lack of information to work on.

**Reply by T145** [reply]
http://www.zytrax.com/books/dns/ch8/dname.html
